api-schema/src/main/java/org/stellar/anchor/api/sep/sep31/Sep31DepositInfo.java

```diff
@@ -0,0 +1,12 @@
+package org.stellar.anchor.api.sep.sep31;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class Sep31DepositInfo {
```

Comment:
@marcelosalloum class `Sep31DepositInfo` seems to belong to Circle implementation and not part of the SEP-31 API. I think this class should belong to `platform` sub-project unless the deposit info is required for all SEP-31 implementations. 

If that's the case, we do not need to define the generator interface. We can make the generator part of the `JdbcSep31TransactionStore` where we update the deposit information.

Reply:
I don't think it should be part of `JdbcSep31TransactionStore` because when using Circle we need to reach an external API, and API calls don't really fit inside database entities.